src/api/todos.ts

```diff
@@ -0,0 +1,22 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const addTodo = (userId: number, data: Todo | null) => {
+  return client.post<Todo>(`/todos?userId=${userId}`, data);
+};
+
+export const removeTodo = (id: number) => {
+  return client.delete(`/todos/${id}`);
+};
+
+export const updateTodo = (id: number, property: string | boolean) => {
+  const nameOfProperty = typeof property === 'string' ? 'title' : 'completed';
```

Comment:
Instead of determining the nameOfProperty based on the type of the property, it's better to explicitly specify what properties can be updated in the function parameters.

Reply:
Or you can just sent an entire object to update